installer/templates/phx_assets/app.css

```
@@ -1,6 +1,15 @@
-@import "tailwindcss/base";
-@import "tailwindcss/components";
-@import "tailwindcss/utilities";
+/* See the Tailwind configuration guide for advanced usage
+   https://tailwindcss.com/docs/configuration */
+
+@import "tailwindcss";
```

It seems Tailwind is looking into deps directory by default. My suggestion is to do:

```suggestion
@import "tailwindcss" source(none);
@source "../js";
@source "../../lib/<%= @lib_web_name || @app_name %>";
```